vars/RedHat.yml

```diff
@@ -28,3 +28,6 @@ __php_opcache_conf_filename: 10-opcache.ini
 __php_fpm_daemon: php-fpm
 __php_fpm_conf_path: "/etc/fpm"
 __php_fpm_pool_conf_path: "/etc/php-fpm.d/www.conf"
+
+__php_fpm_pool_user: apache
```

Comment:
@valdars : indeed, for me the problem only occurs when `php_enable_php_fpm` is `true`. That starts the php-fpm service, which fails when the pool user/groups are not defined. 

@geerlingguy :
Strange, I don't have the `www-data` user on my fresh CentOS 7 instances. What apache package do you use ?

Since the php RPM depends on httpd, the `apache` user will be available on most php installs, and seemed a good default to me. I intend to pair PHP with nginx, and it seems this should work even if they don't use the same users.

What default value would you suggest ?